specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/logic.json

```diff
@@ -6508,6 +6523,10 @@
           "$ref": "#/definitions/FlowEndpointsConfiguration",
           "description": "The endpoints configuration."
         },
+        "accessControl": {
+          "$ref": "#/definitions/FlowAccessControlConfiguration",
+          "description": "The access control configuration."
+        },
         "sku": {
```

Comment:
The SKU should be removed as it is no longer supported. Rohitha confirmed this with us last week.

(WorkflowHostingPlanNotSupportedForApiVersion) The workflow 'atest' uses a hosting plan. Usage of a hosting plan is deprecated since API version '2016-06-01'

Reply:
As per discussion offline, Sku is supported property for existing legacy workflow with Sku name != ResourceSkuName.Consumption. So i will leave it as it is.